src/SourceBuild/content/eng/build.targets

```diff
@@ -0,0 +1,52 @@
+<Project>
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetValidArchiveItems" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetClosestOfficialSdk" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="FindArchiveDiffs" />
+
+  <Target Name="ReportSdkArchiveDiffs"
+          AfterTargets="Build"
+          DependsOnTargets="DetermineSourceBuiltSdkVersion"
+          Condition="'$(ShortStack)' != 'true' AND '$(PortableBuild)' == 'true'" >
+          <!-- ShortStack doesn't produce an SDK, and GetClosestOfficialSdk doesn't support finding non-portable SDKs -->
+
+      <Message Text="Comparing built SDK against closest official build"
+               Importance="High"/>
+
+      <GetValidArchiveItems ArchiveItems="@(SdkTarballItem)"
+                            ArchiveName="dotnet-sdk">
+        <Output TaskParameter="ValidArchiveItems"
+                ItemName="_BuiltSdkArchivePath"/>
+      </GetValidArchiveItems>
+
+      <!-- There should only be 1 SDK archive -->
+      <Error Text="Multiple valid dotnet-sdk archives found."
+             Condition="'@(_BuiltSdkArchivePath->Count())' != '1'" />
+
+      <GetClosestOfficialSdk BuiltArchivePath="@(_BuiltSdkArchivePath)">
+        <Output TaskParameter="ClosestOfficialArchivePath"
+                PropertyName="_ClosestOfficialSdkPath" />
+      </GetClosestOfficialSdk>
+
+      <FindArchiveDiffs BaselineArchive="@(_BuiltSdkArchivePath)"
```

Comment:
I think it would make more sense to create a text file with the diff (i.e. a csv) that we then analyze with a reporting tool to build a dashboard, than adding messages to stdout.

Reply:
Do you think a csv or a `git diff` style `.diff` file would make more sense?

Reply:
Csv is fine as it would be great to then build a grafana or similar dashboard that takes the Csv as an input.